source/_components/media_player.dlna_dmr.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "DLNA DMR"
+description: "Instructions on how to integrate a DLNA DMR device into Home Assistant."
+date: 2018-06-01 14:27
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dlna.png
+ha_category: Media Player
+featured: false
+ha_release: 0.71
+ha_iot_class: "Local Push"
+---
+
+The `dlna_dmr` platform allows you to control a [DLNA Digital Media Renderer](https://www.dlna.org/), such as DLNA enabled TVs or radios.
+
+To add a DLNA DMR device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: dlna_dmr
+    url: http://192.168.0.10:9197/dmr
+    name: "TV living room"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.